### PR TITLE
Create basic UI

### PR DESCRIPTION
**Pull request by Timowest12**
Created a basic UI with all functionality
- Add a book
- Add a person or teacher
- list books and persons

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+end
+
+def list_rentals_by_person_id
+  print 'ID of person: '
+  id = gets.chomp.to_i
+
+  puts 'Rentals:'
+  @rentals.each do |rental|
+    puts "Date: #{rental.date}, Book '#{rental.book.title}' by #{rental.book.author}" if rental.person.id == id
+  end
```
Comment (Dipeshtwis) [inline main.rb:165]: - [ ] Currently, when checking for a list of rentals, After entering the person `id` it's throwing an error. Please fix this.


Screenshot from 2022-02-06 00-26-17

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Timowest12, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

- [ ] README file should have steps on how to run the project. Please mention the command `cd OOP-library` and `ruby main.rb` after cloning the project step in the README file so that any user can run the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (Timowest12) [reply main.rb:165]: fixed

Comment (DeliceLydia) [reply main.rb:165]: well addressed this

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @Timowest12, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**